### PR TITLE
Audit radio input

### PR DESCRIPTION
**Pull request by laurakwhit**
### Screenshots (if applicable) 📸 
| Before (Dark) | After (Dark) | Before (Light) | After (Light) |
|--|--|--|--|
|Screenshot 2024-04-05 at 2 09 10 PM| Screenshot 2024-04-05 at 2 06 29 PM |  Screenshot 2024-04-05 at 2 09 36 PM | Screenshot 2024-04-05 at 2 15 10 PM|
| n/a |Screenshot 2024-04-05 at 2 05 44 PM | n/a | Screenshot 2024-04-05 at 2 10 38 PM|

### Design Considerations 🎨 
Screenshot 2024-04-05 at 1 42 15 PM

### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### CODE REVIEW

File: src/lib/theme/plugin.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export const variables = {
   '--color-subtle': rgb(getColor('slate', 950)),
   '--color-brand': rgb(getColor('indigo', 800)),
   '--color-interactive': rgb(getColor('indigo', 600)),
+  '--color-focus': rgb(getColor('indigo', 500)),
```
Comment (laurakwhit) [inline src/lib/theme/plugin.ts:13]: Open to ideas on the naming and the use of this color 💡

File: src/lib/holocene/radio-input/radio-input.stories.svelte [hunk 2]
```diff
@@ -22,35 +22,55 @@
...
-<Story name="radio input">
+<Story
+  name="radio input"
+  args={{
+    disabled: false,
+  }}
+  template="radio-input"
+/>
+
+<Template id="radio-input" let:args>
```
Comment (GraceGardner) [inline src/lib/holocene/radio-input/radio-input.stories.svelte:33]: Just curious, what does Template do? TIA

**Review by GraceGardner — APPROVED** [review]
Looks good! I like the name focus.

File: src/lib/holocene/radio-input/radio-input.stories.svelte (hunk 2 above)
Comment (laurakwhit) [reply src/lib/holocene/radio-input/radio-input.stories.svelte:33]: The template pattern is part of Storybook Args, which allows stories to get dynamic data via input arguments. So in this case we can toggle `disabled` and see the UI change in the story.

**Pull request merged**